safenode/src/domain/storage/address/mod.rs

```diff
@@ -22,23 +24,26 @@ use serde::{Deserialize, Serialize};
 use xor_name::XorName;
 
 /// An address of data on the network.
-#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize, Debug)]
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize, Debug)]
 pub enum DataAddress {
     ///
     Chunk(ChunkAddress),
     ///
     Register(RegisterAddress),
     ///
     Spend(DbcAddress),
+    /// The new general identifier for a items in the network.
```

Comment:
for an item in the network

However I feel this is bit tricky (as `item` covers Chunk/Register/Spend) which may cause some misuse that:
on one side we use address of `Chunk`, `Register` , `Spend`, but on other side use `Network` key.

Shall we `force` to use just one via type safe?

Reply:
Yeah, I wasn't quite sure, so felt this is an intermediate step.

Thinking about it now... if good from get go or can wait..

Reply:
The differentiation in DataAddress is maybe not necessary since we anyway always know what the action (query/cmd) is by the name of it..

Maybe on the benefit side is that it's a bit clearer in code. But I don't think there are many places where we could be confused about it.

Reply:
I will defer a potential DataAddress update to a follow up PR.